tools/logicalimager/tsk_logical_imager.cpp

```diff
@@ -257,23 +257,23 @@ static bool hasTskLogicalImager() {
 
 /*
 * matchCallback - The function is passed into the LogicalImagerConfiguration.
-*                 It is called when a file matches a rule. Depending on the matchedRuleInfo setting,
+*                 It is called when a file matches a rule. Depending on the matchResult setting,
 *                 this function may extract the matched file and alert the user.
 *
-* @param matchedRuleInfo The MatchedRuleInfo
+* @param matchResult The MatchedRuleInfo
 * @param fs_file TSK_FS_FILE that matches the rule
 * @param path Path of the file
 *
 * @returns TSK_IMG_TYPE_ENUM TSK_OK if callback has no error
 */
-static TSK_RETVAL_ENUM matchCallback(const MatchedRuleInfo *matchedRuleInfo, TSK_FS_FILE *fs_file, const char *path) {
+static TSK_RETVAL_ENUM matchCallback(const MatchedRuleInfo *matchResult, TSK_FS_FILE *fs_file, const char *path) {
```

Comment:
Why is there so much renaming in this PR that should only be fixing memory leaks?

Reply:
I am getting lots of merge conflicts because of the refactor jira. I am still seeing lots of memory leaks. I am fixing these.
The renaming is probably from the refactoring. I'll fix them later.